qiskit_algorithms/amplitude_estimators/ae.py

```diff
@@ -462,7 +403,7 @@ def compute_confidence_interval(
             NotImplementedError: If the confidence interval method `kind` is not implemented.
         """
         # if statevector simulator the estimate is exact
-        if isinstance(result.circuit_results, (list, np.ndarray)):
+        if exact:
```

Comment:
I can;t comment in the line 402 above but I guess it has been around for some time (since it was in Aqua) and left unchanged.

>AquaError: If 'mle' is not in self._ret.keys() (i.e. `run` was not called yet).